sql/cagg_utils.sql

```diff
@@ -27,3 +27,12 @@ CREATE OR REPLACE FUNCTION _timescaledb_functions.cagg_get_bucket_function_info(
     -- fixed or variable sized bucket
     OUT bucket_fixed_width BOOLEAN
 ) RETURNS RECORD AS '@MODULE_PATHNAME@', 'ts_continuous_agg_get_bucket_function_info' LANGUAGE C STRICT VOLATILE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_functions.cagg_parse_invalidation_record(
+    invalidation_record BYTEA,
+    OUT hypertable_id REGCLASS,
```

Comment:
Nit: I would say just relid. Maybe we'd like to track changes for other tables in the future.

```suggestion
    OUT relid REGCLASS,
```

Reply:
I'm using `hypertable_relid` elsewhere, but seems to have forgot this one. Fixing.